src/test/java/org/mockitousage/stubbing/StubbingWithThrowablesTest.java

```diff
@@ -39,48 +52,40 @@ public void shouldStubWithThrowable() throws Exception {
         IllegalArgumentException expected = new IllegalArgumentException("thrown by mock");
         when(mock.add("throw")).thenThrow(expected);
 
-        try {
-            mock.add("throw");
-            fail();
-        } catch (IllegalArgumentException e) {
-            assertEquals(expected, e);
-        }
+        exception.expect(sameInstance(expected));
```

Comment:
Missing equals here

Reply:
Can you give me some more infos? We expect the instance to be thrown that was passed to the stubbing, right?

Reply:
Sorry, this one is correct. I misread